.github/workflows/release-python.yml

```diff
@@ -0,0 +1,36 @@
+name: Python Package
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
This would mean we try to publish the Python package for every tagged commit? Which would fail because we keep trying to re-publish the version 1.1.1 of the Python package? The Python package has changed roughly once in the last few years, while I keep updating the actual Ghidra plugin. I'm not sure if there is a good way to uncouple this. Maybe a different tag for Python releases?